tests/tests.py

```diff
@@ -2119,41 +2119,41 @@ def is_prime(n):
     def test_scans(self):
         "Run scans, with and without agents"
 
-        # Check simple runscans
-        res, out, _ = RUN(["ivre", "runscans", "--output", "Test", "--test",
-                           "2"])
-        self.assertEqual(res, 0)
-        self.assertTrue(b'\nRead address 127.0.0.1\n' in out)
-        self.assertTrue(b'\nRead address 127.0.0.2\n' in out)
-        res = RUN(["ivre", "runscans", "--network", "127.0.0.1/31"])[0]
-        self.assertEqual(res, 0)
-        fdesc = tempfile.NamedTemporaryFile(delete=False)
-        fdesc.writelines(("127.0.0.%d\n" % i).encode() for i in range(2, 4))
-        fdesc.close()
-        res = RUN(["ivre", "runscans", "--file", fdesc.name, "--output",
-                   "XMLFork"])[0]
-        self.assertEqual(res, 0)
-        os.unlink(fdesc.name)
-        res = RUN(["ivre", "runscans", "--range", "127.0.0.4", "127.0.0.5",
-                   "--output", "XMLFull"])[0]
-        self.assertEqual(res, 0)
-        count = sum(len(walk_elt[2]) for walk_elt in os.walk('scans'))
-        self.assertEqual(count, 9)
-
-        # Generate a command line
-        res = RUN(["ivre", "runscans", "--output", "CommandLine"])[0]
-        self.assertEqual(res, 0)
-
-        # Scan using an agent
-        with AgentScanner(self) as agent:
-            agent.scan(["--test", "2"])
-
-        # Count the results
-        count = sum(len(walk_elt[2]) for walk_elt in os.walk('output/'))
-        self.assertEqual(count, 2)
-
-        # Clean
-        shutil.rmtree('output')
+        # # Check simple runscans
```

Comment:
Why did you comment those lines?

Reply:
Sorry my mistake, was working on the tests and I did'nt realize my pull request will update here to...
I though i was working on my fork.... I think I'll close this pull request and put a proper one once done...